### PR TITLE
Feature/initial release

### PR DESCRIPTION
**Pull request by tomme87**
Initial release of oidc client bundle

### CODE REVIEW

File: Token/AbstractToken.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  protected $rawToken;
+
+  /**
+   * @param JWT    $JWT
+   * @param string $rawToken
+   *
+   * @return static
+   */
+  public static function createFromJwt(JWT $JWT, string $rawToken): self {
+    $accessToken = new static();
```
Comment (erichjsonfosse) [inline Token/AbstractToken.php]: Variable should be named $token in the abstract class

**Review by erichjsonfosse — CHANGES_REQUESTED** [review]
As per conversation: Helpers folder should be moved to root

File: composer.json [hunk 2]
```diff
@@ -14,7 +14,11 @@
     "require": {
         "php": "^7.3",
         "symfony/framework-bundle": "^4.4|^5.0",
-        "halloverden/symfony-oidc-contracts": "^1.0"
+        "halloverden/symfony-oidc-contracts": "^1.0",
+        "halloverden/symfony-security": "@dev",
```
Comment (tomme87) [inline composer.json]: Shoud not be @dev wait for new relase of halloverden/symfony-security and fix.

Comment (tomme87) [reply composer.json]: Never mind. 1.4.0 should be fine for this.

File: composer.json [hunk 3]
```diff
@@ -14,7 +14,11 @@
     "require": {
         "php": "^7.3",
         "symfony/framework-bundle": "^4.4|^5.0",
-        "halloverden/symfony-oidc-contracts": "^1.0"
+        "halloverden/symfony-oidc-contracts": "^1.0",
+        "halloverden/symfony-security": "^1.4",
```
Comment (tomme87) [inline composer.json]: might as well wait for halloverden/symfony-security 2.0 and use that before releasing this.

**Pull request merged**